pkg/infradriver/infradriver.go

```diff
@@ -32,8 +32,8 @@ type InfraDriver interface {
 	//GetClusterRootfs /var/lib/sealer/data/${clusterName}/rootfs
 	GetClusterRootfs() string
 
-	// GetImageMountDir /var/lib/sealer/data/${clusterName}/mount
-	GetImageMountDir() string
+	// GetImageMountDir /var/lib/sealer/data/${clusterName}/mount/${os_arch_variant}
+	GetImageMountDir(platform v1.Platform) string
```

Comment:
Please add more description about the reason you add this pull request, otherwise others could not follow you within few minutes. Here I feel quite strange that we add arch information at a local path. 😕

Reply:
in oder to support to run heterogeneous clusters , i add arch information at a local mounted path to distribute different images.